documentation/userguide/docs/datasets.md

```diff
@@ -257,6 +257,35 @@ Catalog and then click on Synchronize as we did in step 3.
 - Or directly, <a href="https://github.com/aws-samples/aws-glue-samples/tree/master/utilities/Hive_metastore_migration">migrating from Hive Metastore.</a>
 - there are more for sure :)
 
+#### (Going Further) Creating Filters on Tables
+
+Additionally, dataset owners can create column-level or row-level filters on their dataset tables to more granularly restrict data access when sharing with other teams. 
+
+To do so dataset owners can navigate to the **Filters** Tab for a given table and select **Add New Filter**:
+
+![dataset_table_filter](pictures/datasets/dataset_table_filter.png#zoom#shadow)
+
```

Comment:
Small suggestion: " you have the choice to create a column filter to filter by select colummns to include only" has a small typo and is a bit difficult to read -->"you have the choice to create a column filter to include a subset of selected columns" or something like that

Reply:
Updated the wording to be more clear